utils/response.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import (
+	"encoding/json"
+	"log"
+)
+
+// RespMsg is general response body
+type RespMsg struct {
```

Comment:
How about just `Response` as an identifier? I think it's more readable plus `Message` is already used as an attribute